templates/bundles/SyliusShopBundle/Homepage/index.html.twig

```diff
@@ -0,0 +1,72 @@
+{% extends '@SyliusShop/layout.html.twig' %}
+
+{% block content %}
+    <div class="homepage">
+        <div class="ui hidden divider"></div>
+            {{ bitbag_cms_render_media('homepage_banner') }}
+        <div class="ui hidden divider"></div>
+
+        <h2 class="ui center aligned huge header">{{ 'sylius.ui.latest_products'|trans }}</h2>
+        <div class="ui hidden divider"></div>
+
+        <div {{ sylius_test_html_attribute('latest-products') }}>
+            {{ render(url('sylius_shop_partial_product_index_latest', {'count': 3, 'template': '@SyliusShop/Homepage/_list.html.twig'})) }}
+        </div>
+        <div class="ui hidden divider"></div>
+
+        <h2 class="ui center aligned huge header">{{ 'sylius.homepage.brace_yoursel_winter_is_coming'|trans }}</h2>
+        <div class="ui hidden divider"></div>
+
+        {% include '@SyliusShop/Homepage/_carousel.html.twig' %}
```

Comment:
I think we should be able to just customize the way block is rendered in this case 

ref. https://github.com/BitBagCommerce/SyliusCmsPlugin/blob/master/doc/blocks.md#general-usage